gebru/roster.json

```diff
@@ -109,7 +109,9 @@
       "materials": [
         { 
           "type": "cv"
-        }
+        },
+        "type": "video",
+        "url": "8xEfNT1708Y"
```

Comment:
@rsho-create 
You are missing the curly brackets

Reply:
Thanks, added the curly brackets